app/src/debugTools/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugNotificationManager.kt

```diff
@@ -62,7 +72,7 @@ class DebugNotificationManager @Inject constructor(
                 0,
                 "Low Battery",
                 PendingIntent.getBroadcast(
-                    context,
+                    service,
                     2,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a089e65afe6044fe8b2c4847ffd33998441cf079/app/src/debugTools/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugNotificationManager.kt#L76"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>